nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -18,7 +18,7 @@
 
 ## Backward Incompatibilities {#sec-release-24.11-incompatibilities}
 
-- `gnome.nautilus` was renamed to `nautilus` and is now expected to be managed through [](#opt-programs.nautilus.enable) NixOS option.
+- `gnome.nautilus` was renamed to `nautilus` and is now expected to be managed through [](#opt-programs.nautilus.enable) NixOS option. Additionally, binary Nautilus extensions will no longer be picked up from [](#opt-environment.systemPackages), they need to be installed through [](#opt-programs.nautilus.extensions). (This does not apply to `nautilus-python` scripts, which should be installed through as before.)
```

Comment:
BTW what does "install as before" mean?

Reply:
Just that no change is necessary.

Reply:
> Just that no change is necessary.

If there were no instructions before how to install python extensions for nautilus, it is a bit inconsistent to write this...

Reply:
In the absence of specific documentation, standard methods of installation (e.g. `environment.systemPackages` or user profile) and [upstream instructions](https://gitlab.gnome.org/GNOME/nautilus-python/#running-extensions) apply.

And the backwards incompatibilities section applies to people that already had the software installed anyway.

Reply:
It is as if you are saying that all of the documentation that I added in https://github.com/NixOS/nixpkgs/pull/240780 is trivially inferred from reading the upstream instructions, and or simply using `environment.systemPackages`.. Did I understand you correctly?

Reply:
Except for the `NAUTILUS_4_EXTENSIONS_DIR` and the GSettings behaviour, yes.

- The environment variable, I do not want to expose.
- The GSettings behaviour is not extension specific, and the whole section needs rewrite anyway.
  - We need to mention dconf NixOS module
  - Ideally remove GSettings overrides and/or implement GSettings module based on the dconf one
  - Mention that we do not install GSettings globally as you mention (I have to keep linking people to https://github.com/NixOS/nixpkgs/issues/33277)